src/components/parameters.js

```diff
@@ -1000,10 +989,16 @@ const Parameters = ({ showParameters, hideParameters, user }) => {
                     <TabPanel value={tabIndex} index={3}>
                         <NetworkParameters />
                     </TabPanel>
-                    <Grid item xs={12}>
+                    <Grid container className={classes.controlItem}>
+                        {tabIndex === 2 ? (
```

Comment:
avoid double ternary operator experssions

Replace with 
{tabIndex === 2 && (...)}
{tabIndex === 3 && (...)}

Or replace with a function MakeDefaultValuesButton that returns the button depending on the index